gym_ignition/base/runtime.py

```diff
@@ -50,3 +50,16 @@ def unwrapped(self):
             The gym.Env environment of the Runtime.
         """
         return self
+
+    @abc.abstractmethod
+    def timestamp(self) -> float:
+        """
+        Return the timestamp associated to the execution of the environment.
+
+        In real-time environments, the timestamp is the time read from the host system.
+        In simulated environments, the timestamp is the simulated time, which might not
+        match the real-time in the case of RTF different than 1.
+
+        Returns:
+            A float indicating the current environment timestamp
```

Comment:
What are the units of the timestamp?

Reply:
At this early stage I didn't want to put any constraints. It is implementation dependent, think about gazebo and yarp for instance.

Reply:
Ok, even if I think that both in YARP and in Gazebo the unit of the timestamps is units (when expressed as a scalar).

Reply:
The timestamp could be an incremental counter, a time that starts from zero, or an unix timestamp for instance. All these are just a scalar representing a timestamp.